app/dashboard/src/components/JSONSchemaInput.tsx

```diff
@@ -139,7 +139,7 @@ export default function JSONSchemaInput(props: JSONSchemaInputProps) {
                 value={typeof value === 'number' ? value : ''}
                 size={1}
                 className={twMerge(
-                  'focus-child min-6- text40 w-60 grow rounded-input border-0.5 bg-transparent px-input-x read-only:read-only',
+                  'focus-child min-6- text40 w-full grow rounded-input border-0.5 bg-transparent px-input-x read-only:read-only',
```

Comment:
oops, i think `min-6- text40` is broken... we probably want to check in blame for the original correct code

Reply:
![CleanShot 2024-09-16 at 10 11 55@2x](https://github.com/user-attachments/assets/fefdcf18-f824-46d2-bce6-eb786896f22c)

I think it was a bad regex: https://github.com/enso-org/enso/pull/10715/files#diff-714d58b65c74fc059da856df96d3e22e551c9aa5de3396fd7de6a52a50b120b3R157

Reply:
hmm. we may want to enable the invalid tailwind classes lint. thoughts?

Reply:
agree